Header header
geometry_msgs/Pose camera_pose
geometry_msgs/Point[] centres3d
geometry_msgs/Point[] centres2d
